packages/playground/src/components/funds_card.vue

```diff
@@ -25,31 +25,29 @@
 </template>
 
 <script lang="ts">
-import type { GridClient } from "@threefold/grid_client";
 import { ref } from "vue";
 
 import { manual } from "@/utils/manual";
 
 import { useProfileManagerController } from "../components/profile_manager_controller.vue";
-import { useGrid } from "../stores";
+import { useProfileManager } from "../stores";
 import { createCustomToast, ToastType } from "../utils/custom_toast";
-import { updateGrid } from "../utils/grid";
+import { getGrid } from "../utils/grid";
```

Comment:
Why did we switch from `getGrid` to `useGrid` in [this PR](https://github.com/threefoldtech/tfgrid-sdk-ts/pull/2563) and then switch back here?

Reply:
We were supposed to use the store to reduce the number of grid instances in the Dashboard. This produced multiple issues like when listing deployments or when using the get TFT. That's why we're reverting back these changes.

Reply:
What kind of issues?

Reply:
It broke the functionalites.

Reply:
You can revert back to the mentioned PR. it has a log with the comments for more clarification.

Reply:
okay, I will approve but we need to take the know what should we use the `gridStore` or `getGrid`